Solutions/Marain.UserNotifications.Abstractions/Marain/NotificationTemplates/CommunicationTemplates/EmailTemplate.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <copyright file="EmailTemplate.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.NotificationTemplates.CommunicationTemplates
+{
+    /// <summary>
+    /// Email Object.
+    /// </summary>
+    public class EmailTemplate : ICommunicationTemplate
+    {
+        /// <summary>
+        /// The content type that will be used when serializing/deserializing.
+        /// </summary>
+        public const string RegisteredContentType = "application/vnd.marain.usernotifications.notificationtemplate.emailtemplate.v1";
+
+        /// <inheritdoc/>
+        public string? NotificationType { get; set; }
+
+        /// <inheritdoc/>
+        public string? ETag { get; set; }
+
+        /// <summary>
+        /// Gets or sets the body of the email.
+        /// </summary>
+        public string? Body { get; set; }
+
+        /// <summary>
+        /// Gets or sets the subject of the email.
+        /// </summary>
+        public string? Subject { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether the email is important or not.
+        /// </summary>
+        public bool? Important { get; set; }
```

Comment:
As states above EmailTemplate would be reworked when work commenses on sending emails through a delivery channel.

Reply:
Ok. Since we can't make a proper decision about where it goes right now, let's get rid of the property completely. That way, when it (or something equivalent) gets added back in during implementation of the first email delivery channel, we won't forget to discuss it.

Reply:
I have removed all instances of `important` from email templates for now. Yes agreed, we can think more on this when we are near to actually implement the delivery channel to send an email.